src/main/resources/templates/grpc/java/rpc_java_mediator.vm

```diff
@@ -284,10 +284,9 @@ public class ${capitalizedKey}Mediator extends AbstractConnector {
            #end
 
            String jsonOutput = JsonFormat.printer().preservingProtoFieldNames().print(response);
-           org.apache.axis2.context.MessageContext axisMsgCtx = ((Axis2MessageContext) context).getAxis2MessageContext();
-           JsonUtil.getNewJsonPayload(axisMsgCtx, jsonOutput, true, true);
-           axisMsgCtx.setProperty(Constants.Configuration.MESSAGE_TYPE, "application/json");
-           axisMsgCtx.setProperty(Constants.Configuration.CONTENT_TYPE, "application/json");
+           String responseVariable = (String) getParameter(context, "responseVariable");
```

Comment:
If we extend the AbstractConnectorOperation class, then this might not be needed? WDYT? is there any limitation in that?

Reply:
Yeah, we can use this AbstractConnectorOperation and updated the code.